presto-docs/src/main/sphinx/release/release-0.279.rst

```diff
@@ -2,6 +2,10 @@
 Release 0.279
 =============
 
+.. warning::
```

Comment:
I don't think this needs a warning.  Adjusted queue size is a metric used for operations, it's not a correctness issue.